pom.xml

```diff
@@ -1032,50 +1086,96 @@
 
 
 	<dependencies>
-	
-		<!-- For GPU backend
-			Use org.mystic:mavenized-jcuda until Alan puts org.jcuda:*
-		 -->
-		<dependency>
-			<groupId>org.mystic</groupId>
-            <artifactId>mavenized-jcuda</artifactId>
-            <version>0.7.5b</version>
-            <type>jar</type>
-            <scope>provided</scope>
-            <exclusions>
-		        <exclusion>
-		            <groupId>*</groupId>
-		            <artifactId>*</artifactId>
-		        </exclusion>
-		    </exclusions>
-		</dependency>
-		<!-- Since there is no mvn repo for jcuda
+
 		<dependency>
 			<groupId>org.jcuda</groupId>
 			<artifactId>jcuda</artifactId>
-			<version>0.7.5b</version>
-			<scope>provided</scope>
+			<version>${jcuda.version}</version>
```

Comment:
@nakul02  can you please mark all jcuda related artifacts as "provided" dependency ? The PR increases the size of SystemML.jar from 4.7 MB to 6.4 MB.